.editorconfig

```diff
@@ -7,7 +7,7 @@ charset = utf-8
 insert_final_newline = true
 trim_trailing_whitespace = true
 
-[*.{yml,twig,php}]
+[*.{yaml,yml,twig,php}]
```

Comment:
why a second?

Reply:
Otherwise the editor would not use the options for the new symfony config flies which ends with .yaml instead of .yml

Or what do you mean?